src/server/server.js

```diff
@@ -10,78 +10,75 @@ const {
 
 class Server {
```

Comment:
This whole file has been changed in the latest updates with updated comments, you'll need to pull the latest from develop into your local branch.